### PR TITLE
📖 docs(amp-date-display): add README for bento component

### PR DESCRIPTION
**Pull request by kvchari**
adds readme for bento component 

continues: https://github.com/ampproject/amphtml/pull/35875 and #35874

### CODE REVIEW

File: extensions/amp-date-display/1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: The Bento Date Display component displays time data that you can render in your page.
+experiental: true
+bento: true
```
Comment (caroqliu) [inline extensions/amp-date-display/1.0/README.md:8]: these two annotations should not be needed

**Review by caroqliu — APPROVED** [review]

File: extensions/amp-date-display/1.0/README.md (hunk 1 above)
Comment (kvchari) [reply extensions/amp-date-display/1.0/README.md]: updated

**Comment by kvchari** [comment]
consolidated changes from the date-countdown PR here

**Comment by caroqliu** [comment]
> consolidated changes from the date-countdown PR here

ahh, this was totally unclear instructions from my part - I was suggesting to consolidate `amp-inline-gallery` and `amp-stream-gallery` since both PRs modified `amp-inline-gallery`, and `amp-stream-gallery` is a closely related component.

Feel free to keep them separate or consolidate as you see fit otherwise. But, regardless, this and the other are both approved.

**Pull request merged**